<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1394</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelWidth">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>690</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>690</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Location of cursor :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>690</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>690</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Size of Image :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>690</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeight">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>690</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>690</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>690</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImg_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Image</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>661</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1039</width>
       <height>659</height>
      </rect>
     </property>
     <widget class="QScrollArea" name="scrollAreaMainWindow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1391</width>
        <height>951</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1389</width>
         <height>949</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="inputDepth_btn">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Input depth for a set of points</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="numberOfPoints_text">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfPoints_label">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>90</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the number of poitns to be captured :</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="initPointSelection_btn">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>150</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select points clicked
on the Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterDepth_btn">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter depth for 
selected points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_toTest">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>310</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_toTest2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>430</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>680</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Select next set
 of points (for another building)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="skyGrassCheck">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>40</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Input for Sky or Front Grass</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1394</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
